internal/postprocessor/main_test.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"flag"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"cloud.google.com/go/internal/gapicgen/git"
+	"github.com/google/go-cmp/cmp"
+)
+
+var googleapisDir string
```

Comment:
This appears not to be used and there is a hardcoded local value in the test.

Reply:
Removed hard-coded value.